### PR TITLE
Add v1.16.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Write content for each headline / feature
- [x] Update contributors list

### CODE REVIEW

**Comment by pmuens** [comment]
@worldsoup @DavidWells the blog post is now ready for a review.

I added the content for all the pending chapters and updated the list with the contributors.

**Comment by worldsoup** [comment]
schedule(06/22/2017 6:00 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **06/22/2017 at 6:00am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/203#issuecomment-310205445)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

File: posts/2017-06-22-serverless-v1.16.0.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+          method: POST
+          path: greet
+          private: true
+```
+
+**Note:** Please make sure to enable usage plan support in your [AWS API Gateway console](https://console.aws.amazon.com/apigateway/home) by following the steps in the [announcement blog post](https://aws.amazon.com/de/blogs/aws/new-usage-plans-for-amazon-api-gateway/).
+
+### Significantly reduced time to deploy by excluding development dependencies
+
+The Serverless Frameworks `package` plugin includes a sophisticated zipping utility which gives you control over the `.zip` file creation process through config parameters such as `exclude` or `include`.
```
Comment (ryanmurakami) [inline posts/2017-06-22-serverless-v1.16.0.md]: Frameworks -> Framework's

File: posts/2017-06-22-serverless-v1.16.0.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+Most of the time the serverless functions you work on rely on different 3rd party packages whether they're production relevant packages or packages used to streamline the development process (e.g. test runners, offline utilities, etc.).
```
Comment (ryanmurakami) [inline posts/2017-06-22-serverless-v1.16.0.md]: Do you want to capitalize 'serverless' here?

**Comment by pmuens** [comment]
Thanks for the review @ryanmurakami! 👍 

Just pushed some updates.

**Comment by pmuens** [comment]
Thank for sending the video @ryanmurakami 🎉 

I just added it to the post! 🙌

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**